tests/load-tests/ci-scripts/max-concurrency/collect-results.sh

```diff
@@ -75,6 +75,9 @@ ${csv_delim}ClusterPodCountAvg\
 ${csv_delim}ClusterPVCInUseAvg\
 ${csv_delim}ClusterPipelineRunCountAvg\
 ${csv_delim}ClusterPipelineWorkqueueDepthAvg" \
+${csv_delim}ClusterPipelineScheduleFirstPodAvg" \
+${csv_delim}ClusterTaskrunThrottledByNodeResourcesAvg" \
+${csv_delim}ClusterTaskRunThrottledByDefinedQuotaAvg" \
```

Comment:
```suggestion
${csv_delim}ClusterPipelineWorkqueueDepthAvg\
${csv_delim}ClusterPipelineScheduleFirstPodAvg\
${csv_delim}ClusterTaskrunThrottledByNodeResourcesAvg\
${csv_delim}ClusterTaskRunThrottledByDefinedQuotaAvg" \
```
@gabemontero The quote only needs to go at the end :)

Reply:
ah thanks @pmacik :-)